app/src/lib/shells/win32.ts

```diff
@@ -0,0 +1,88 @@
+import { spawn } from 'child_process'
+import * as Path from 'path'
+import { assertNever } from '../fatal-error'
+import { readRegistryKeySafe } from '../registry'
+import { IFoundShell } from './found-shell'
+
+export enum Shell {
+  Cmd = 'cmd',
```

Comment:
`cmd.exe` is the executable, but Command Prompt is the friendly name. Here's how it looks in the start menu when I search for it:

<img width="400" src="https://user-images.githubusercontent.com/359239/29437378-8033f686-83f3-11e7-8a05-48fb459846cc.png">